enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+
+#### Single-node Deployments or MicroShift
+
+The ClusterVersion operator will honor the change management field just as in a standalone profile. If those profiles
+have a MachineConfigPool, material changes the node could be controlled with a change management policy
+in that resource.
+
+#### OCM Managed Profiles
+OpenShift Cluster Manager (OCM) should expose a user interface allowing users to manage their change management policy.
+Standard Fleet clusters will expose the option to configure the MaintenanceSchedule strategy - including
+only permit and exclude times.
+
+- Service Delivery will reserve the right to disable this strategy for emergency corrective actions.
+- Service Delivery should constrain permit & exclude configurations based on their internal policies. For example, customers may be forced to enable permissive windows which amount to at least 6 hours a month.
+
+### Implementation Details/Notes/Constraints
+
+#### ChangeManagement Stanza
+The change management stanza will be introduced into ClusterVersion and MachineConfigPool (for standalone profiles)
+and HostedCluster and NodePool (for HCP profiles). The structure of the stanza is:
+
+```yaml
+spec:
+  changeManagement:
+    # The active strategy for change management (unless disabled by disabledUntil).
+    strategy: <strategy-name|Disabled|null>
+    
+    # If set to true or a future date, the effective change management strategy is Disabled. Date 
+    # must be RFC3339. 
+    disabledUntil: <bool|date|null>
+    
+    # If set to true or a future date, all strategies other than Disabled are paused. Date 
+    # must be RFC3339. 
+    pausedUntil: <bool|date|null>    
+    
+    # If a strategy needs additional configuration information, it can read a 
+    # key bearing its name in the config stanza.
+    config:
+       <strategy-name>:
+         ...configuration policy for the strategy...
+```
+
+#### MaintenanceSchedule Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: MaintenanceSchedule
+    config:
+       maintenanceSchedule:
+        # Specifies a reoccurring permissive window. 
+        permit:  
+          # RRULEs (https://www.rfc-editor.org/rfc/rfc5545#section-3.3.10) are commonly used 
+          # for calendar management metadata. Only a subset of the RFC is supported. If
+          # unset, all dates are permitted and only exclude constrains permissive windows.
+          recurrence: <rrule|null>
+          # Given the identification of a date by an RRULE, at what time (relative to timezoneOffset) can the 
+          # permissive window begin. "00:00" if unset.
+          startTime: <time-of-day|null>
+          # Given the identification of a date by an RRULE, at what time (relative to timezoneOffset) should the 
+          # permissive window end. "23:59:59" if unset.
+          endTime: <time-of-day|null>
+        
+        # Excluded date ranges override RRULE selections.
+        exclude:
```

Comment:
Recommend allowing a datetime, not just date, for the fromDate and untilDate to give full control to.  Else the configuration is very coarse and may result in customer frustration.  For example, if an exclusion period is through 12PM UTC on a Sunday and updates can only be done on a weekend, with this configuration it's not possible to use the remaining 12 hours on Sunday to apply updates.